Backend/dhall-configs/dev/dynamic-offer-driver-app.dhall

```diff
@@ -115,12 +109,6 @@ let driverPoolCfg =
       , driverRequestCountLimit = +3
       }
 
-let intelligentPoolConfig =
-      { minQuotesToQualifyForIntelligentPool = +5
-      , minQuotesToQualifyForIntelligentPoolWindowOption =
-        { period = +24, periodType = common.periodType.Hours }
-      }
-
 let overrideDriverPoolCfg =
```

Comment:
remove this too.

Reply:
Remove `PoolSortingType`, `driverPoolBatchesCfg`, `sendSearchRequestJobCfg` and all other configs which are not being used.